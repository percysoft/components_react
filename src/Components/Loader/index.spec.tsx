import * as React from 'react';
import { shallow, mount } from 'enzyme';
import { Loader } from './index';

describe('Loader Component', () => {
  describe('Render snapshot', () => {
    test('Render correctly Loader with number parameter', () => {
      const wrapper = mount(
          <Loader heightLoader={100}/>);
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
