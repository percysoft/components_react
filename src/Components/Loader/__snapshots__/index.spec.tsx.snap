// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component Render snapshot Render correctly Loader with number parameter 1`] = `"<div class=\\"b-loader\\" style=\\"width: 100px; height: 100px;\\"><div class=\\"b-loader_spinner\\"><svg class=\\"b-loader_spinner__svg\\" viewBox=\\"0 0 120 120\\" fill=\\"transparent\\"><circle r=\\"42\\" cx=\\"60\\" cy=\\"60\\" stroke=\\"#333\\" stroke-width=\\"6\\" stroke-dasharray=\\"2 9.5\\"></circle><linearGradient id=\\"gradient-0-60\\"><stop offset=\\"0%\\" stop-color=\\"#fff\\" stop-opacity=\\"0.5\\"></stop><stop offset=\\"100%\\" stop-color=\\"#fff\\" stop-opacity=\\"0\\"></stop></linearGradient><rect x=\\"0\\" y=\\"60\\" width=\\"60\\" height=\\"60\\" fill=\\"url(#gradient-0-60)\\" class=\\"b-loader_spinner__rect\\"></rect><linearGradient id=\\"gradient-0-0\\"><stop offset=\\"0%\\" stop-color=\\"#fff\\" stop-opacity=\\"0.5\\"></stop><stop offset=\\"100%\\" stop-color=\\"#fff\\" stop-opacity=\\"0.75\\"></stop></linearGradient><rect x=\\"0\\" y=\\"0\\" width=\\"60\\" height=\\"60\\" fill=\\"url(#gradient-0-0)\\" class=\\"b-loader_spinner__rect\\"></rect><linearGradient id=\\"gradient-60-0\\"><stop offset=\\"0%\\" stop-color=\\"#fff\\" stop-opacity=\\"0.75\\"></stop><stop offset=\\"100%\\" stop-color=\\"#fff\\" stop-opacity=\\"0.9\\"></stop></linearGradient><rect x=\\"60\\" y=\\"0\\" width=\\"60\\" height=\\"60\\" fill=\\"url(#gradient-60-0)\\" class=\\"b-loader_spinner__rect\\"></rect><linearGradient id=\\"gradient-60-60\\"><stop offset=\\"0%\\" stop-color=\\"#fff\\" stop-opacity=\\"0.9\\"></stop><stop offset=\\"100%\\" stop-color=\\"#fff\\" stop-opacity=\\"1\\"></stop></linearGradient><rect x=\\"60\\" y=\\"60\\" width=\\"60\\" height=\\"60\\" fill=\\"url(#gradient-60-60)\\" class=\\"b-loader_spinner__rect\\"></rect><linearGradient id=\\"arc-gradient\\"><stop offset=\\"0%\\" stop-opacity=\\"0\\"></stop><stop offset=\\"49.99%\\" stop-opacity=\\"0\\"></stop><stop offset=\\"50%\\" stop-color=\\"#BBB\\"></stop><stop offset=\\"100%\\" stop-color=\\"#333\\"></stop></linearGradient><circle r=\\"36\\" cx=\\"60\\" cy=\\"60\\" fill=\\"transparent\\" stroke=\\"url(#arc-gradient)\\" stroke-width=\\"2\\" class=\\"b-loader_spinner__circle\\"></circle></svg></div></div>"`;
